package.json

```diff
@@ -6,12 +6,15 @@
   "license": "AGPL-3.0",
   "private": true,
   "scripts": {
-    "build": "ncc build src/index.js"
```

Comment:
No longer bundling it?

Reply:
not used, reverted bundling long time ago. maybe we can do this later.

I'll switch the warning to an error in a few weeks, then some time later i'll add back the bundling

Reply:
I don't mind. Just thought it was a change that might need extra testing